### PR TITLE
Added error pages handlers

### PR DESCRIPTION
**Pull request by BrainIsDead**
The changes made in this PR include:
- Added BadRequestErrorView, ForbiddenErrorView, NotFoundErrorView, and InternalServerErrorView classes to base/views.py
- Modified signal_documentation/urls.py to include the error views as handlers for HTTP errors
- Added 400.html, 403.html, 404.html, and 500.html templates to the templates directory
- Added error.html template to the templates directory as a base for all error pages.

### CODE REVIEW

File: src/templates/400.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{% load i18n %}
+
+{% block content %}
+<div class="container">
+
+    <section class="section error-404 min-vh-100 d-flex flex-column align-items-center justify-content-center">
+      <h1>400</h1>
+      <h2>Bad Request.</h2>
+      <a class="btn" href="{% url 'signals' %}">Back to home</a>
+      <img src="{% static 'assets/img/not-found.svg' %}" class="img-fluid py-5" alt="Bad Request.">
```
Comment (krivard) [inline src/templates/400.html:13]: What does the `assets/img/not-found.svg` image look like?

**Review by krivard — COMMENTED** [review]
Just questions for now

File: src/templates/400.html (hunk 1 above)
Comment (BrainIsDead) [reply src/templates/400.html:13]: image
image
image
image

**Review by krivard — COMMENTED** [review]
This PR now includes almost a thousand files; was that intentional?

File: src/templates/error.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+
+  <title>Not Found 404</title>
+  <meta content="Not Found 404" name="description">
+  <meta content="HTML, CSS, JavaScript" name="keywords">
```
[the remark this answers is not in the document]
Comment (BrainIsDead) [reply src/templates/error.html]: made meta empty

**Comment by BrainIsDead** [comment]
> This PR now includes almost a thousand files; was that intentional?



> This PR now includes almost a thousand files; was that intentional?

I haven't pushed assets directory before, it includes files needed for bootstrap

**Review by krivard — APPROVED** [review]
👍

**Pull request merged**